analyses/sv-analysis/01-process-sv-file.R

```diff
@@ -0,0 +1,187 @@
+# Yang Yang 2019
+# This script is for generating sv file in shatterseek-read/signature-read input format
+# 
+# input files are:
+# 1.independent-specimens.wgs.primary-plus.tsv
+# this file is used to choose independent specimens
+# 2. pbta-sv-manta.tsv
+# this file is the original sv file generated by manta call
+# 
+# output files are:
+# sv files for each specimens
+# 
+# output file format:
+# chrom1, pos1, chrom2, pos2, SVtype, strand1, strand2, alt1, alt2, homolen, inserlen
+# chrom and pos are breakpoint location
+# SVtype is structral variation type, they are DUP, DEL, INV and TRA 
+# strand1 and strand2 will be used in shatterseek analysis
+# alt1 and alt2 were used to infer strand， you can ignore them
+# homolen and inserlen will be used in signature analysis
+
+
+library(stringr)   
+
+## ===================== Load  Independent Specimen List =====================
```

Comment:
One way to deal with the directories is to use the `rprojroot` to make it easy to run this from wherever. Here's an example of someone using it in another module of the project:
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/54a8a1ae9d91881aed0360df1ac47df8d940c139/analyses/transcriptomic-dimension-reduction/scripts/run-dimension-reduction.R#L50
You can set the root directory of the project like so:
```r
root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
```
And then to read in the independent specimen list you would do:

```r
independent_specimen_list <-  read.table(file.path(root_dir, "data/independent-specimens.wgs.primary-plus.tsv"), header = TRUE, sep = "\t")
```

This should then work no matter where someone called the script from.